rasa/shared/core/domain.py

```diff
@@ -1537,7 +1537,7 @@ def intent_config(self, intent_name: Text) -> Dict[Text, Any]:
         return self.intent_properties.get(intent_name, {})
 
     @rasa.shared.utils.common.lazy_property
-    def intents(self):
+    def intents(self) -> Text:
```

Comment:
as it's sorted I'd expect it to be a collection?
```suggestion
    def intents(self) -> List[Text]:
```